api/core/database/models/api/bot/user.py

```diff
@@ -59,11 +58,8 @@ def validate_discriminator(
 
     @validates("roles")
     def validate_roles(self, _key: str, roles: list[int]) -> Union[list[int], NoReturn]:
-        """Raise ValueError if the provided role(s) is negative or non-existent."""
-        session = SessionLocal()
+        """Raise ValueError if the provided role(s) is negative."""
         for role in roles:
             if role < 0:
                 raise ValueError("Role IDs cannot be negative")
-            if not session.query(Role).filter_by(id=role).first():
-                raise ValueError(f"Role with ID {role} does not exist")
```

Comment:
I don't understand why you removed this check 🤔

Reply:
It was removed due to the switch to async sqlalchemy. The validations will be performed elsewhere, I'll make sure to implement the mentioned validation later on.